ccd-client/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+  id 'net.ltgt.apt' version '0.10'
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  compile project(':ccd-adapter')
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-feign', version: '1.3.5.RELEASE'
+  compile group: 'io.github.openfeign', name: 'feign-httpclient', version: '9.5.0'
+
+  compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.16.18'
+  apt group: 'org.projectlombok', name: 'lombok', version: '1.16.18'
```

Comment:
Please extract `lombok` version to a variable and us that instead of copy-pasting the value.

Reply:
module is moved out in it's own repo.